lib/redix.ex

```diff
@@ -1,21 +1,41 @@
 defmodule GuardianRedis.Redix do
   @moduledoc """
-    Redix module
+  Redix module.
+
+  ### Configuration
+  Configuration options are the same as in `Redix.start_link/1`, except for
+  `pool_size`, `name_prefix`, and `name`.
+
+  - `pool_size` - determines how many Redix connections are created; defaults to `1`
+  - `name_prefix` - the prefix used in `name`; defaults to `"redix"`
+
+  The `name` option is overwritten based on the given `name_prefix` and `pool_size`.
+  In the case of the example below, `name` would be `:g_redis_0` for the first
+  Redis connection, and `:g_redis_1` for the second.
+
+  ```
+  config :guardian_redis, :redis,
+    host: "127.0.0.1",
+    port: 6379,
+    pool_size: 2,
+    name_prefix: "g_redis"
+  ```
   """
-  @default_redis_host "127.0.0.1"
-  @default_redis_port 6379
+  @default_redis_name_prefix "redix"
```

Comment:
what do you think of default prefix be 'guardian_redis' ?

Reply:
That would be a breaking change, but since there are two breaking changes already, I don't see the harm of it, and it does indeed seem like a better prefix :)

Reply:
ok nvm did it just now :)

Reply:
hm tests are failing now... so back to tonight it is hah

Reply:
ok fixed, was an actual bug as I missed one spot when adding the `name_prefix`, yikes. so a good idea to change the name indeed!

Reply:
perfect! thank you for this!